core/src/main/scala/cats/Show.scala

```diff
@@ -120,8 +117,12 @@ object Show extends ScalaVersionSpecificShowInstances with ShowInstances {
 private[cats] trait ShowInstances extends cats.instances.NTupleShowInstances with ShowInstances0 {
   implicit def catsShowForFiniteDuration: Show[FiniteDuration] =
     cats.instances.finiteDuration.catsStdShowForFiniteDurationUnambiguous
+
+  implicit def catsShowForSortedSet[A: Show]: Show[SortedSet[A]] = cats.instances.sortedSet.catsStdShowForSortedSet[A]
```

Comment:
This could've stayed in the old position

Reply:
If I move catsShowForSortedSet back where it was in the Show companion object, I get an ambiguous implicit error on `ContravariantShow[BitSet]`

> [...]\cats\tests\shared\src\test\scala\cats\tests\ShowSuite.scala:92:15
> ambiguous implicit values:
>  both method catsShowForBitSet in object Show of type cats.Show[scala.collection.immutable.BitSet]
>  and method catsShowForSortedSet in object Show of type [A](implicit evidence$8: cats.Show[A]): cats.Show[scala.collection.immutable.SortedSet[A]]
>  match expected type cats.Show.ContravariantShow[scala.collection.immutable.BitSet]
>     implicitly[ContravariantShow[BitSet]]

Reply:
Ah yes, `BitSet` is a subtype of `SortedSet`, good catch 👍